### PR TITLE
#312: Get rid of tens of billing/shipping id calls

### PR DESCRIPTION
**Pull request by bonkey**
I know it might be better to use simple string ids at least in lowest call in APIClient, but I think devs would work with existing addresses anyway.

### CODE REVIEW

File: AtlasUI/AtlasUI/AtlasCheckout.swift [hunk 1]
```diff
@@ -56,22 +56,26 @@ public class AtlasCheckout: LocalizerProviderType {
         viewController.presentViewController(navigationController, animated: true, completion: nil)
     }
 
-    func prepareCheckoutViewModel(selectedArticleUnit: SelectedArticleUnit, checkoutViewModel: CheckoutViewModel? = nil,
+    func createCheckout(fromModel checkoutViewModel: CheckoutViewModel,
```
Comment (ahmed-shehata) [inline AtlasUI/AtlasUI/AtlasCheckout.swift]: Why change the function name? This does not return a checkout.

**Comment by bonkey** [comment]
@ahmed-shehata forgive my blindness but for my eyes it does

File: AtlasUI/AtlasUI/AtlasCheckout.swift (hunk 1 above)
Comment (bonkey) [inline AtlasUI/AtlasUI/AtlasCheckout.swift]: Ahc, now I see

**Comment by bonkey** [comment]
👍

**Comment by artdaw** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by artdaw** [comment]
👍

**Pull request merged**